README.md

```diff
@@ -146,6 +146,140 @@ Module blocks could be transpiled to either data URLs, or to a module in a separ
 
 This proposal only allows anonymous module blocks. There are other proposals for named module *bundles* (with URLs corresponding to the specifier of each JS module), including "[JS Module Bundles]" proposal, and [Web Bundles](https://www.ietf.org/id/draft-yasskin-wpack-bundled-exchanges-03.html). Note that there are significant privacy issues to solve with bundling to permit ad blockers; see [concerns from Brave](https://brave.com/webbundles-harmful-to-content-blocking-security-tools-and-the-open-web/).
 
+## FAQs
+
+### Should we really allow creation of workers using module blocks?
+
+In my opinion: Yes. The requirement that workers are in a separate file is one of the most prominent pieces of feedback about why workers are hard to adopt. That’s why so many resort to Blob URLs or Data URLs, bringing along all kinds of difficulties, especially relating to paths and CSP. The risk here is that people start spawning a lot workers without regards to their cost, but I think the benefits of lowering the barrier to workers as an important performance primitive outweigh the risks.
+
+### Can you close over variables? Can you reference values outside the module block?
+
+No. Just like a separate file containing a ES module, you can only reference the global scope and import other modules.
+
+### Can you _statically_ import other modules?
+
+Yes. Just like with a separate file. This is completely valid:
+
+```js
+const m = module {
+  import myApiWrapper from "./api-wrapper.js";
+
+  await someTopLevelAwaitLogic();
+}
+```
+
+### What about a nice shorthand for modules that just contain one function?
+
+Something like this?
+
+```js
+const m = module (x) => x**2;
+
+// equivalent to
+
+const m = module {
+  export default (x) => x**2;
+}
+```
+
+It is unclear whether this will actually be a common pattern, as it not only requires the module to have exactly one function, but also that it does not need any static imports. For now we plan to leave the MVP without such a shorthand and observe what kind of usage patterns emerge. If import-free single-function modules become a frequently recurring pattern, a module shorthand seems like a good idea to add.
+
+### Can Module Blocks help with bundling?
+
+Mostly no. In _some_ scenarios Module Blocks can be sufficient to bundle:
```

Comment:
Suggested wording: "Mostly no. At first glance, it may look like Module Blocks could provide a bundling format for simple scenarios like this:"